src/pages/SubmissionDetails.js

```diff
@@ -104,8 +106,7 @@ export const SubmissionDetails = ({ submissionId }) => {
       }
     }
 
-    await refresh()
-    loadRunSetData(signal).then(runSet => runSet && loadMethodsData(runSet.method_version_id))
+    await refresh().then(() => loadMethodsData(undefined))
```

Comment:
This might look like a very dramatic change, but if you look at the actual calls in the browser, you'll notice that `runSet.method_version_id` was always undefined... presumably because `loadRunSetData` is actually returning an array, and the array doesn't have a `method_version_id` field